.tekton/pull-request.yaml

```diff
@@ -117,11 +117,14 @@ spec:
               imagePullPolicy: Always
               args: [
                 "--ginkgo.label-filter=build-templates-e2e",
-                "--ginkgo.progress",
                 "--ginkgo.v",
                 "--ginkgo.no-color"
               ]
               env:
+              - name: APP_SUFFIX
+                value: "{{ pull_request_number }}"
+              - name: COMPONENT_REPO_URLS
+                value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep"
```

Comment:
Is it running multiple test? From log I can see only `PipelineRun test-component-znig-5lcp5 Status.Conditions.Reason: Running`

Reply:
yeah the logging is not great :/ but it's there

```
[build-service-suite Build service E2E tests] HACBS pipelines should eventually finish successfully for component with source URL https://github.com/redhat-appstudio-qe/devfile-sample-python-basic [build, HACBS, build-templates-e2e]
/github.com/redhat-appstudio/e2e-tests/tests/build/build.go:466
  PipelineRun test-component-znig-5lcp5 Status.Conditions.Reason: Running
  ...
  PipelineRun test-component-znig-5lcp5 Status.Conditions.Reason: Succeeded
...
[build-service-suite Build service E2E tests] HACBS pipelines should eventually finish successfully for component with source URL https://github.com/redhat-appstudio-qe/retrodep [build, HACBS, build-templates-e2e]
/github.com/redhat-appstudio/e2e-tests/tests/build/build.go:466
  PipelineRun test-component-znig-5lcp5 Status.Conditions.Reason: Succeeded
```

Reply:
oh I see, for some reason the component name is the same, let me check the code

Reply:
fix: https://github.com/redhat-appstudio/e2e-tests/pull/341

Reply:
issue fixed in the linked PR
```
[build-service-suite Build service E2E tests] HACBS pipelines should eventually finish successfully for component with source URL https://github.com/redhat-appstudio-qe/devfile-sample-python-basic [build, HACBS, build-templates-e2e]
/github.com/redhat-appstudio/e2e-tests/tests/build/build.go:468
  PipelineRun test-component-lcpb-jvzn2 Status.Conditions.Reason: Running
  ...
  PipelineRun test-component-lcpb-jvzn2 Status.Conditions.Reason: Completed
...
[build-service-suite Build service E2E tests] HACBS pipelines should eventually finish successfully for component with source URL https://github.com/redhat-appstudio-qe/retrodep [build, HACBS, build-templates-e2e]
/github.com/redhat-appstudio/e2e-tests/tests/build/build.go:468
  PipelineRun test-component-cstn-lwzt7 Status.Conditions.Reason: Running
  PipelineRun test-component-cstn-lwzt7 Status.Conditions.Reason: Succeeded
```